proto/internal/uber/cadence/sharddistributor/v1/service.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+syntax = "proto3";
+
+package uber.cadence.sharddistributor.v1;
+
+option go_package = "github.com/uber/cadence/.gen/proto/sharddistributor/v1;sharddistributorv1";
+
+// ShardDistributorAPI is used to query shard distributor for the current owner of a shard
+service ShardDistributorAPI {
+
+  // GetShardOwner returns the owner of a specific shard
+  rpc GetShardOwner(GetShardOwnerRequest) returns (GetShardOwnerResponse);
+}
+
+message GetShardOwnerRequest {
+  string shard_key = 1;
+  string namespace = 2;
+}
+
+message GetShardOwnerResponse {
+  string owner = 1;
```

Comment:
What is owner here? host:port or just host or some other identity? Please make it clear in the comments in proto file so API is self descriptive.